plugins_src/import_export/wpc_svg_path.erl

```diff
@@ -225,4 +758,2941 @@ getcedges([#pathop{opkind=pcurveto,x1=X1,y1=Y1,x2=X2,y2=Y2,x3=X,y3=Y}|Ops],
     getcedges(Ops,{X,Y},First,
         [#cedge{vs=Prev,cp1={X1,Y1},cp2={X2,Y2},ve={X,Y}}|Acc]);
 getcedges([_|_],_,_,_) ->
-    [].	% funny path (probably moveto in middle), so return nothing
+    [].    % funny path (probably moveto in middle), so return nothing
+
+
+%% Apply transforms within SVG coordinate system to path
+%%
+apply_path_transform(OSubpaths, TA) ->
+    [Path#path{ops=[ begin
+        {X1,Y1} = vs_m3x2({X1_0,Y1_0},TA),
+        {X2,Y2} = vs_m3x2({X2_0,Y2_0},TA),
+        {X3,Y3} = vs_m3x2({X3_0,Y3_0},TA),
+        PathOp#pathop{x1=X1,y1=Y1,x2=X2,y2=Y2,x3=X3,y3=Y3}
+       end
+        || #pathop{x1=X1_0,y1=Y1_0,x2=X2_0,y2=Y2_0,x3=X3_0,y3=Y3_0}=PathOp <- Ops_0]}
+        || #path{ops=Ops_0}=Path <- OSubpaths].      
+-spec vs_m3x2({float(),float()}, 'none' | m3x2()) -> {float(), float()}.
+vs_m3x2({X,Y}, none) ->
+    {X,Y};
+vs_m3x2({X,Y}, {matrix, M00,M10,M20, M01,M11,M21}) ->
+    { M00*X + M20*Y + M11,
+      M10*Y + M01*Y + M21 }.
+
+%%
+%% New xmerl parsing section for SVG files.
+%%
+
+read_svg_content(Bin_0, FullFilename, ShortFilename) ->
+    EF = {event_fun, fun svg_tok/3},
+    ES = {event_state, #svgtk{}},
+    {ok, Bin_1} = svg_change_prolog(Bin_0),
+    case xmerl_sax_parser:stream(Bin_1, [EF,ES]) of
+        {ok, #svgtk{list=List,units=DocUnits_S,docsz=DocSz,viewbox=ViewBox}=_Es, _} ->
+            DocUnits = unit_atom(DocUnits_S),
+            CommandList_0 = lists:reverse(List),
+            {ok, {Definitions, CommandList_1}} =
+                path_definitions(CommandList_0,
+                    #document_s{fullfilename=FullFilename,units=DocUnits,docsz=DocSz,viewbox=ViewBox}),
+            {ok, CommandList_2} = retr_d_svg_paths(Definitions, CommandList_1, ShortFilename),
+            {ok, CommandList_3} = assign_transforms(CommandList_2),
+            {ok, CommandList_3, #document_s{fullfilename=FullFilename,units=DocUnits,docsz=DocSz,viewbox=ViewBox}};
+        {Error, {_,_,Line}, Reason, _ET, _St} ->
+            io:format("SVG:~p: ERROR: ~p:~p~n", [Line, Error, Reason]),
```

Comment:
This debug printout is ok, i.e. for errors cases debug printouts are ok but for the normal case it should
be quiet.